src/sfall_config.h

```diff
@@ -80,6 +80,7 @@ namespace fallout {
 #define SFALL_CONFIG_PIPBOY_AVAILABLE_AT_GAMESTART "PipBoyAvailableAtGameStart"
 #define SFALL_CONFIG_USE_WALK_DISTANCE "UseWalkDistance"
 #define SFALL_CONFIG_AUTO_OPEN_DOORS "AutoOpenDoors"
+#define SFALL_CONFIG_GAPLESS_MUSIC "GaplessMusic"
```

Comment:
Is this an actual sfall config, or something new?

Reply:
sfall has this feature always on since [v4.3.1](https://github.com/sfall-team/sfall/blob/master/CHANGELOG.md#431).
> Added the ability to continuously play the music when moving to another map that uses the same music

Reply:
Thanks! How did you handle the loading "static" sound?  Just disable it entirely?

Reply:
If you mean the wind sound, it plays if the player is in the world map (i.e. entering a random encounter map) or when the map music isn't the same.
https://github.com/sfall-team/sfall/blob/master/sfall/Modules/MiscPatches.cpp#L843